static/js/periodShortcuts.js

```diff
@@ -0,0 +1,44 @@
+var _ = require('ep_etherpad-lite/static/js/underscore');
+var utils = require('./utils');
+
+var DOT_KEY_CODE = 190;
+var SCENE_EXTENSIONS = ["EXT.", "INT.", "E/I.", "I/E."];
+
+exports.createSceneIfHasSceneExtension = function (rep, attributeManager, editorInfo) {
+  var lineNumber = rep.selStart[0];
+  var $line = getLineFromLineNumber(lineNumber)
+  var lineTextIsASceneExtension = textOfLineIsSceneExtension($line);
+  var lineIsAction = isLineAction($line);
+
+  if (lineTextIsASceneExtension && lineIsAction && hasNotSelection(rep)) {
```

Comment:
uma desvantagem de pegar os valores das 3 variáveis antes é que vc executa as 3 funções, mas nem sempre isso é necessário -- se a 1a variável já for `false`, as outras poderiam ser ignoradas. O melhor nesses casos é usar direto as funções dentro do if: `if (textOfLineIsSceneExtension($line) && isLineAction($line) && ...)`